### PR TITLE
Fixed some language mistakes in 'About the code' doc page

### PR DESCRIPTION
**Pull request by iyovcheva**

### CODE REVIEW

**Comment by bostko** [comment]
I think all fixes are correct.
LGTM

File: docs/guide/dev/code/index.md [hunk 1]
```diff
@@ -39,11 +39,11 @@ This document is intended to help people become familiar with the codebase.
 
 Brooklyn is split into the following projects and sub-projects:
 
-* **``camp``**: the components for a server which speaks the CAMP REST API and which understands the CAMP YAML plan language
+* **``camp``**: the components for a server which speaks with the CAMP REST API and understands the CAMP YAML plan language
```
Comment (andreaturli) [inline docs/guide/dev/code/index.md]: I think `the` was more appropriate then `with`

Comment (mikezaccardo) [inline docs/guide/dev/code/index.md]: "... which speaks *to* the CAMP REST API"

Comment (hzbarcea) [inline docs/guide/dev/code/index.md]: Or even "speaks the CAMP REST API"

For ``utils`` I would also just say 'lower level utilities' (drop the first 2 words).

My $0.02...

**Comment by ahgittin** [comment]
This fixes some useful things, merging.  The minor semantic changes can be done separately.

**Pull request merged**